src/components/Carousel/CarouselItem.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import {
+  COLOUR_SECONDARY_TEXT,
+  COLOUR_SECONDARY_BACKGROUND,
+  REM_PX,
+} from '../../theme'
+
+const CarouselItem = ({ children }) => {
+  return (
+    <div className="carousel__item">
+      {children}
+      <style jsx>{`
+        .carousel__item {
+          background-color: ${COLOUR_SECONDARY_BACKGROUND};
+          color: ${COLOUR_SECONDARY_TEXT};
+          padding: ${REM_PX * 5}px ${REM_PX * 4}px;
```

Comment:
Should the padding be using the gutter_px? I'm honestly not sure.
If they should be rem, should we instead use `5rem`, `4rem` etc?